src/components/DataInput/DataInput.cy.tsx

```diff
@@ -213,5 +213,11 @@ describe("Data Input Components", () => {
             cy.get("input[value='c']").click();
             cy.get("@onChangeSpy").should("have.been.calledWith", "c");
         });
+
+        it("Add error message to form", () => {
+            cy.mount(<FreeFormTextInput label="FreeForm" error={true} helperText="Error" />);
+            cy.get(".Mui-error").contains("Error");
+            cy.get(".Mui-error").should("have.css", "color", hexToRgb("#d32f2f"));
```

Comment:
is this a stylistic thing from mui? maybe set at the theme level? or is it 'default red'?